docs/docs/30-administration/100-external-configuration-api.md

```diff
@@ -0,0 +1,106 @@
+# External Configuration API
+
+To provide additional management and preprocessing capabilities for pipeline configurations Woodpecker supports an HTTP api which can be enabled to call an external config service.
+Before the run or restart of any pipeline Woodpecker will make a POST request to an external HTTP api sending the current repository, build information and all current config files retrieved from the repository. The external api can then send back new pipeline configurations that will be used immediately or respond with `HTTP 204` to tell the system to use the existing configuration.
+
+Every request sent by Woodpecker is signed using a http-signature using the provided secret from `WOODPECKER_CONFIG_SERVICE_SECRET`. This way the external api can verify the authenticity request from the Woodpecker instance.
+
+A simplistic example configuration service can be found here: [https://github.com/s00500/woodpecker-config-service](https://github.com/s00500/woodpecker-config-service)
```

Comment:
@s00500 I have created a repo inside the woodpecker org and added you as maintainer.

```suggestion
A simplistic example configuration service can be found here: [https://github.com/woodpecker-ci/example-config-service](https://github.com/woodpecker-ci/example-config-service)
```

Reply:
✅ 
That's great, I adapted the docs, will do a final check on it and fix potential bugs